app/templates/gentelella/users/events/_table.html

```diff
@@ -1,6 +1,23 @@
 {% macro events_table(events) %}
-    <div class="table-responsive">
-        <table class="table table-striped">
+
+<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
+<script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
+<script type = "text/javascript" >
+	 $('#dodo').DataTable({
```

Comment:
@niranjan94 I meant that I was trying to reduce the number of elements appearing. So, was just using hit and try so as to reduce . when I saw that there was a reduce by 1 , thought to notify you. that whether I was going in the right direction or not .